apis/test_config.py

```diff
@@ -90,6 +89,31 @@ def name(self):
     return f'v{self.version}-{self.cores}'
 
 
+# TODO(ranran): We may want to use GKE in the future.
```

Comment:
Sorry for the confusion if that happens. Could we move this comment to `def name(self):`? so that I know we need to update it.

Reply:
There are several `def name(self):`, which `def name(self):` do you mean here?

Reply:
For this comment, could you help remind me if we want to move your GPU implementation to use GKE? or we need a separate implementation like `TpuGkeTest`? 

If it's former, it makes sense to add the comment here, but you may be the assignee? If the later, it will be a new feature request. I don't think you need to add a TODO here. Let me know your thoughts!

Reply:
Prefer remove the TODO item. Unless we have a solid motivation to move to GKE.